srcpkgs/lsp-plugins/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.1.30
+revision=1
+build_style=gnu-makefile
+make_build_args="VERBOSE=1"
```

Comment:
you know that you can test it on your own computer, right? or are the issues CI specific? also you can push only commits that should be in this PR.

Reply:
@paper42 of course I test on my computer first. But I don't want to install cross toolchains for all possible architectures, so when I want to test for something like `armv6-musl`, I do it on CI.

Reply:
I've removed parallel tests commit because it finally builds fine for all targets.